src/components/TokenClaimDialog.tsx

```diff
@@ -301,29 +328,45 @@ export default function TokenClaimDialog({
 }
 
 function ClaimListItem({
-  title,
-  amount,
   claimCallback,
   status,
+  items,
+  title,
 }: {
-  title: string
-  amount: BigNumber
+  title?: string
   claimCallback: () => void
+  items: [string, BigNumber][]
   status?: STATUSES
 }): ReactElement {
   const { t } = useTranslation()
-  const formattedAmount = commify(formatBNToString(amount, 18, 2))
-  const disabled =
-    status === STATUSES.PENDING ||
-    status === STATUSES.SUCCESS ||
-    amount.lt(BigNumber.from(10).pow(16)) // don't let anyone try to claim less than 0.01 token
+  const disabled = status === STATUSES.PENDING || status === STATUSES.SUCCESS
+  // @dev - our formatting assumes all tokens are 1e18
   return (
     <ListItem>
       <Typography variant="subtitle1" sx={{ flex: 1 }}>
-        {title}
+        {title && (
+          <>
+            {title}
+            <br />
+          </>
+        )}
+        {items.map(([name]) => (
+          <>
+            {name}
+            <br />
+          </>
+        ))}
       </Typography>
       <Typography sx={{ flex: 1 }}>
-        {status === STATUSES.SUCCESS ? 0 : formattedAmount}
+        {title && <br />}
+        {items.map(([, amount]) => (
+          <>
+            {status === STATUSES.SUCCESS
+              ? 0
+              : commify(formatBNToString(amount, 18, 2))}
+            <br />
+          </>
```

Comment:
Something like `<Typography mb={2}>// ternary</Typography>` could also work but this is accomplishing similar.

Reply:
I have this a shot, but I think that line breaks are probably a simpler way of creating a fake table, which is what I'm trying to do here. This UI could definitely be refined